README.md

```diff
@@ -1,3 +1,18 @@
 # sdk-link
+## Configure your auth flow
 
-WIP
+Our Link settings allow you to configure the authorization process based on your data needs and white-label Embedded Link to suit your brand. You can customize these settings in the [Codat Portal](https://app.codat.io/) in [Settings > Auth flow > Link](https://app.codat.io/settings/link-settings).
```

Comment:
```suggestion
Our Link settings allow you to configure the authorization process based on your data needs and label Embedded Link to suit your brand. You can customize these settings in the [Codat Portal](https://app.codat.io/) in [Settings > Auth flow > Link](https://app.codat.io/settings/link-settings).
```

suggesting this because Embedded LInk is already white-label, so the customer cannot white-label it - they can just brand it instead